src/components/Footer/Footer.js

```diff
@@ -10,19 +11,26 @@ import {
   containerGridLight,
   containerIconsDark,
   containerIconsLight,
+  logo,
   mainContainerDark,
   mainContainerLight,
+  name,
+  title,
 } from './Footer.module.scss';
+
 export const Footer = () => {
   const { theme } = useContext(ThemeContext);
   const { t } = useTranslation();
+  const { isDarkTheme } = useContext(ThemeContext);
```

Comment:
use theme function instead, but modify its body so  that lightThemeClassName becomes lightThemeName, this way you can use it in defining theme for src string aswell